ci/docker/runtime_functions.sh

```diff
@@ -192,32 +196,15 @@ build_armv7() {
     popd
 }
 
-build_amzn_linux_cpu() {
-    cd /work/build
-    cmake \
-        -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-        -DCMAKE_C_COMPILER_LAUNCHER=ccache \
-        -DUSE_CUDA=OFF\
-        -DUSE_OPENCV=ON\
-        -DUSE_OPENMP=ON\
-        -DUSE_SIGNAL_HANDLER=ON\
-        -DCMAKE_BUILD_TYPE=RelWithDebInfo\
-        -DUSE_MKL_IF_AVAILABLE=OFF\
-        -DUSE_LAPACK=OFF\
-        -DUSE_DIST_KVSTORE=ON\
-        -G Ninja /work/mxnet
-    ninja -v
-    report_ccache_usage
-}
-
-build_arm64() {
+build_armv8() {
     cmake \
         -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
         -DCMAKE_C_COMPILER_LAUNCHER=ccache \
         -DUSE_CUDA=OFF\
         -DSUPPORT_F16C=OFF\
         -DUSE_OPENCV=OFF\
         -DUSE_OPENMP=OFF\
+        -DUSE_LAPACK=OFF\
```

Comment:
Could you elaborate? Didn't we fix lapack, @lebeg ?

Reply:
Feel free to have a look, this is out of the scope of this PR.

Reply:
No, we didn't. I'm afraid if you'd set this to `USE_LAPACK=OFF` it will not build (openblas needs a symlink liblapack.so to itself). But such a fix is out of the scope of this PR.

Reply:
Excellent, thank you for the confirmation